File: plugin/src/main/java/org/beangle/maven/plugin/patch/Patches.java
package org.beangle.maven.plugin.patch;

public class Patches {

  public static String[] files = { "/org/hibernate/action/internal/CollectionUpdateAction.class",
      "/org/hibernate/collection/spi/PersistentCollection.class",
      "/org/hibernate/collection/internal/AbstractPersistentCollection.class", };

  public static final String PatchFlag = "/META-INF/beangle_patched.txt";

}

File: plugin/src/main/scala/org/beangle/maven/plugin/patch/Patches.scala
package org.beangle.maven.plugin.patch

object Patches {

  var files = List("/org/hibernate/action/internal/CollectionUpdateAction.class", "/org/hibernate/collection/spi/PersistentCollection.class", "/org/hibernate/collection/internal/AbstractPersistentCollection.class")

  val PatchFlag = "/META-INF/beangle_patched.txt"
}
